Userland/Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -526,20 +526,28 @@ FLATTEN_ON_CLANG void Interpreter::run_bytecode(size_t entry_point)
         DISPATCH_NEXT(name);                                                                                                              \
     }
 
+#define HANDLE_INSTRUCTION_WITHOUT_EXCEPTION_CHECK(name)                                    \
```

Comment:
Could we just make this an attribute of the specific `Instruction` class, and use `if constexpr` in the normal macro
two options that come to mind is going by `execute_impl` return type or a value of/presence of a `constexpr` flag
(presence can be checked with a `requires` expression)

Reply:
Oh, that'd be nicer yeah! Great idea :)